source/funkin/ui/story/StoryMenuState.hx

```diff
@@ -336,6 +336,16 @@ class StoryMenuState extends MusicBeatState
           changeDifficulty(0);
         }
 
+        if (FlxG.mouse.wheel < 0)
```

Comment:
this wheel behaviour has things from html and non-html code, check the freeplay code as reference